spec/rspec/mocks_spec.rb

```diff
@@ -132,7 +132,7 @@
         expect(defined?(ValueY)).to be_falsey
       end
 
-      it 'does not allow the stubbed constants to be used after the scope in before(:all)' do
+      it 'does not allow the stubbed constants to be used after the scope in before(:all)', :pending => RSpec::Support::Ruby.jruby_9000? do
```

Comment:
It produces a warning on JRuby for rspec-expectations:

`lib/rspec/matchers/built_in/raise_error.rb:131 warning: too many arguments for format string`

but all it is a `to_s` call, googling this warning resulted in a slew of results pointing to an issue with `sprintf` on JRuby.